scripts/release-update-tilt-docs-repo.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Updates the Tilt docs repo with the latest version info.
+#
+# Usage:
+# scripts/update-docs-tilt-repo.sh $VERSION
+# where VERSION is of the form 0.1.0
+
+set -e
+
+if [[ "$GITHUB_TOKEN" == "" ]]; then
+    echo "Missing GITHUB_TOKEN"
+    exit 1
+fi
+
+VERSION=${1//v/}
+VERSION_PATTERN="[0-9]+\.[0-9]+\.[0-9]+"
```

Comment:
```suggestion
VERSION_PATTERN="^[0-9]+\.[0-9]+\.[0-9]+$"
```

Might as well also prevent, e.g., "a1.2.3b", or "1.2.3-dev", from matching

Reply:
done!